dev/com.ibm.ws.st.core/src/com/ibm/ws/st/core/internal/ConfigurationResourceChangeListener.java

```diff
@@ -266,7 +266,7 @@ protected static void refreshChangesImpl(final List<IFolder> folders, final List
             WebSphereServer ws = WebSphereUtil.getWebSphereServer(server);
             if (ws != null) {
                 if (ws.isLocalSetup())
-                    ws.addLocalConnectorFeature(null);
+                    ws.serverConfigChanged(null);
```

Comment:
This change allows Liberty Web Server extensions to add custom behaviour whenever the server config changes.  The current Liberty behaviour remains the same (the local connector feature is added), but for the build plugin case, the application monitor trigger is set to mbean as well.  The additional, somewhat independent change, is the WebSphereUtil change to get the WebSphereServer.  I think this looks ok.